diff --git a/apps/quarkus-json/src/main/java/org/acme/ExampleResource.java b/apps/quarkus-json/src/main/java/org/acme/ExampleResource.java
index dfb224d..a37b4a3 100644
--- a/apps/quarkus-json/src/main/java/org/acme/ExampleResource.java
+++ b/apps/quarkus-json/src/main/java/org/acme/ExampleResource.java
@@ -1,12 +1,12 @@
 package org.acme;
 
-import javax.ws.rs.Consumes;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
 import java.io.ByteArrayOutputStream;
 import java.math.BigInteger;
 import java.security.MessageDigest;
