diff --git a/apps/quarkus-full-microprofile/pom.xml b/apps/quarkus-full-microprofile/pom.xml
index 42109ed..0af4097 100644
--- a/apps/quarkus-full-microprofile/pom.xml
+++ b/apps/quarkus-full-microprofile/pom.xml
@@ -15,7 +15,7 @@
 
     <properties>
         <quarkus.version>${global.quarkus.version}</quarkus.version>
-        <vertx.auth.jwt.version>4.5.7</vertx.auth.jwt.version>
+        <vertx.auth.jwt.version>5.1.3</vertx.auth.jwt.version>
     </properties>
 
     <dependencyManagement>
@@ -47,10 +47,6 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-jwt</artifactId>
         </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-smallrye-metrics</artifactId>
-        </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-health</artifactId>
diff --git a/apps/quarkus-full-microprofile/src/main/java/com/example/quarkus/metric/MetricController.java b/apps/quarkus-full-microprofile/src/main/java/com/example/quarkus/metric/MetricController.java
deleted file mode 100644
index db4c9fa..0000000
--- a/apps/quarkus-full-microprofile/src/main/java/com/example/quarkus/metric/MetricController.java
+++ /dev/null
@@ -1,49 +0,0 @@
-package com.example.quarkus.metric;
-
-import org.eclipse.microprofile.metrics.Counter;
-import org.eclipse.microprofile.metrics.MetricUnits;
-import org.eclipse.microprofile.metrics.annotation.Gauge;
-import org.eclipse.microprofile.metrics.annotation.Metric;
-import org.eclipse.microprofile.metrics.annotation.Timed;
-
-import jakarta.enterprise.context.ApplicationScoped;
-import jakarta.inject.Inject;
-import jakarta.ws.rs.GET;
-import jakarta.ws.rs.Path;
-import java.util.Random;
-
-@Path("/metric")
-@ApplicationScoped
-public class MetricController {
-
-    @Inject
-    @Metric(name = "endpoint_counter")
-    Counter counter;
-
-    @Path("timed")
-    @Timed(name = "timed-request")
-    @GET
-    public String timedRequest() {
-        // This is just for demo/test proposes!!!
-        int wait = new Random().nextInt(1000);
-        try {
-            Thread.sleep(wait);
-        } catch (InterruptedException e) {
-            // Demo
-            e.printStackTrace();
-        }
-        return "Request is used in statistics, check with the Metrics call.";
-    }
-
-    @Path("increment")
-    @GET
-    public long doIncrement() {
-        counter.inc();
-        return counter.getCount();
-    }
-
-    @Gauge(name = "counter_gauge", unit = MetricUnits.NONE)
-    long getCustomerCount() {
-        return counter.getCount();
-    }
-}
