diff --git a/apps/quarkus-mp-orm-dbs-awt/pom.xml b/apps/quarkus-mp-orm-dbs-awt/pom.xml
index 42ba6f7..18541dd 100644
--- a/apps/quarkus-mp-orm-dbs-awt/pom.xml
+++ b/apps/quarkus-mp-orm-dbs-awt/pom.xml
@@ -22,7 +22,7 @@
         <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
         <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
         <quarkus.platform.version>${quarkus.version}</quarkus.platform.version>
-        <vertx.auth.jwt.version>4.0.3</vertx.auth.jwt.version>
+        <vertx.auth.jwt.version>4.5.7</vertx.auth.jwt.version>
         <apache.pdfbox.version>3.0.0</apache.pdfbox.version>
         <final.name>${final.name}</final.name>
     </properties>
@@ -50,23 +50,23 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy-reactive</artifactId>
+            <artifactId>quarkus-rest</artifactId>
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy-reactive-qute</artifactId>
+            <artifactId>quarkus-rest-qute</artifactId>
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-rest-client-reactive-jsonb</artifactId>
+            <artifactId>quarkus-rest-client-jsonb</artifactId>
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy-reactive-jsonb</artifactId>
+            <artifactId>quarkus-rest-jsonb</artifactId>
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-rest-client-reactive</artifactId>
+            <artifactId>quarkus-rest-client</artifactId>
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
@@ -114,7 +114,7 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-smallrye-opentracing</artifactId>
+            <artifactId>quarkus-opentelemetry</artifactId>
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/awt/AWTApplication.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/awt/AWTApplication.java
index 598beef..718d27a 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/awt/AWTApplication.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/awt/AWTApplication.java
@@ -2,8 +2,8 @@ package quarkus.awt;
 
 import io.quarkus.runtime.Startup;
 
-import javax.annotation.PostConstruct;
-import javax.enterprise.context.ApplicationScoped;
+import jakarta.annotation.PostConstruct;
+import jakarta.enterprise.context.ApplicationScoped;
 import javax.imageio.ImageIO;
 import java.awt.Font;
 import java.awt.FontFormatException;
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/awt/ImageResource.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/awt/ImageResource.java
index 0ef95ff..a644df6 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/awt/ImageResource.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/awt/ImageResource.java
@@ -4,13 +4,13 @@ import org.jboss.resteasy.reactive.MultipartForm;
 import org.jboss.resteasy.reactive.RestForm;
 
 import javax.imageio.ImageIO;
-import javax.inject.Inject;
-import javax.ws.rs.Consumes;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
 import java.awt.AlphaComposite;
 import java.awt.Color;
 import java.awt.Font;
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/ClientController.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/ClientController.java
index 97db779..2e83b4c 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/ClientController.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/ClientController.java
@@ -2,11 +2,11 @@ package quarkus.client;
 
 import org.eclipse.microprofile.rest.client.inject.RestClient;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
 
 @Path("/client")
 @ApplicationScoped
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/Service.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/Service.java
index 66f652f..c728e09 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/Service.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/Service.java
@@ -2,10 +2,10 @@ package quarkus.client;
 
 import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
 
 @RegisterRestClient
 @ApplicationScoped
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/ServiceController.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/ServiceController.java
index 81e055f..b6c5a9d 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/ServiceController.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/client/ServiceController.java
@@ -1,8 +1,8 @@
 package quarkus.client;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
 
 @Path("/client/service")
 public class ServiceController {
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/config/ConfigTestController.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/config/ConfigTestController.java
index daeaef1..1326116 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/config/ConfigTestController.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/config/ConfigTestController.java
@@ -3,10 +3,10 @@ package quarkus.config;
 import org.eclipse.microprofile.config.ConfigProvider;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
 
-import javax.enterprise.context.RequestScoped;
-import javax.inject.Inject;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
 
 @Path("/config")
 @RequestScoped
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/faulttolerance/ResilienceController.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/faulttolerance/ResilienceController.java
index 6058103..8c14e53 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/faulttolerance/ResilienceController.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/faulttolerance/ResilienceController.java
@@ -3,9 +3,9 @@ package quarkus.faulttolerance;
 import org.eclipse.microprofile.faulttolerance.Fallback;
 import org.eclipse.microprofile.faulttolerance.Timeout;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
 
 @Path("/resilience")
 @ApplicationScoped
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/graphql/GraphQLResource.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/graphql/GraphQLResource.java
index db94ada..a943b2c 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/graphql/GraphQLResource.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/graphql/GraphQLResource.java
@@ -9,7 +9,7 @@ import org.eclipse.microprofile.graphql.Mutation;
 import org.eclipse.microprofile.graphql.Name;
 import org.eclipse.microprofile.graphql.Query;
 
-import javax.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.context.ApplicationScoped;
 import java.util.ArrayList;
 import java.util.List;
 
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/health/ServiceLiveHealthCheck.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/health/ServiceLiveHealthCheck.java
index 6c71307..a195727 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/health/ServiceLiveHealthCheck.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/health/ServiceLiveHealthCheck.java
@@ -4,7 +4,7 @@ import org.eclipse.microprofile.health.HealthCheck;
 import org.eclipse.microprofile.health.HealthCheckResponse;
 import org.eclipse.microprofile.health.Liveness;
 
-import javax.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.context.ApplicationScoped;
 
 @Liveness
 @ApplicationScoped
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/health/ServiceReadyHealthCheck.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/health/ServiceReadyHealthCheck.java
index 2d03957..3781c6b 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/health/ServiceReadyHealthCheck.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/health/ServiceReadyHealthCheck.java
@@ -4,7 +4,7 @@ import org.eclipse.microprofile.health.HealthCheck;
 import org.eclipse.microprofile.health.HealthCheckResponse;
 import org.eclipse.microprofile.health.Readiness;
 
-import javax.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.context.ApplicationScoped;
 
 @Readiness
 @ApplicationScoped
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/jwt/JWTResource.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/jwt/JWTResource.java
index 27f5b93..b053c4f 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/jwt/JWTResource.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/jwt/JWTResource.java
@@ -8,13 +8,13 @@ import io.vertx.ext.auth.jwt.JWTAuth;
 import io.vertx.ext.auth.jwt.JWTAuthOptions;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
 
-import javax.annotation.PostConstruct;
-import javax.enterprise.context.ApplicationScoped;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.WebApplicationException;
-import javax.ws.rs.client.ClientBuilder;
-import javax.ws.rs.core.Response;
+import jakarta.annotation.PostConstruct;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.core.Response;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
 import java.util.Objects;
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/jwt/ProtectedResource.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/jwt/ProtectedResource.java
index 56d2a07..252ce0f 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/jwt/ProtectedResource.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/jwt/ProtectedResource.java
@@ -3,11 +3,11 @@ package quarkus.jwt;
 import org.eclipse.microprofile.jwt.Claim;
 import org.eclipse.microprofile.jwt.ClaimValue;
 
-import javax.annotation.security.RolesAllowed;
-import javax.enterprise.context.RequestScoped;
-import javax.inject.Inject;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
+import jakarta.annotation.security.RolesAllowed;
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
 
 @Path("/protected")
 @RequestScoped
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/metric/MetricController.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/metric/MetricController.java
index 9c4d7de..fb7840c 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/metric/MetricController.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/metric/MetricController.java
@@ -6,10 +6,10 @@ import org.eclipse.microprofile.metrics.annotation.Gauge;
 import org.eclipse.microprofile.metrics.annotation.Metric;
 import org.eclipse.microprofile.metrics.annotation.Timed;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
 
 @Path("/metric")
 @ApplicationScoped
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/EntityResource.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/EntityResource.java
index 45c7190..fcf3094 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/EntityResource.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/EntityResource.java
@@ -4,15 +4,15 @@ import io.quarkus.panache.common.Sort;
 import quarkus.orm.db1.DB1Entity;
 import quarkus.orm.db2.DB2Entity;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.transaction.Transactional;
-import javax.ws.rs.Consumes;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.WebApplicationException;
-import javax.ws.rs.core.Response;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.transaction.Transactional;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.core.Response;
 import java.util.List;
 
 @Path("orm/entities")
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/db1/DB1Entity.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/db1/DB1Entity.java
index 0d85436..e177ea2 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/db1/DB1Entity.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/db1/DB1Entity.java
@@ -2,8 +2,8 @@ package quarkus.orm.db1;
 
 import io.quarkus.hibernate.orm.panache.PanacheEntity;
 
-import javax.persistence.Entity;
-import javax.persistence.Table;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
 
 @Entity
 @Table(name = "db1entity")
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/db2/DB2Entity.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/db2/DB2Entity.java
index d2c822a..f57a9eb 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/db2/DB2Entity.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/orm/db2/DB2Entity.java
@@ -2,8 +2,8 @@ package quarkus.orm.db2;
 
 import io.quarkus.hibernate.orm.panache.PanacheEntity;
 
-import javax.persistence.Entity;
-import javax.persistence.Table;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
 
 @Entity
 @Table(name = "db2entity")
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/pdfbox/PDFBoxResource.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/pdfbox/PDFBoxResource.java
index d5b5d8a..9b7f395 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/pdfbox/PDFBoxResource.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/pdfbox/PDFBoxResource.java
@@ -7,12 +7,12 @@ import org.jboss.resteasy.reactive.MultipartForm;
 import org.jboss.resteasy.reactive.RestForm;
 
 import javax.imageio.ImageIO;
-import javax.ws.rs.Consumes;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
 import java.io.ByteArrayOutputStream;
 import java.io.File;
 
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/qute/SomePage.java b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/qute/SomePage.java
index fb080f1..f25ecca 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/qute/SomePage.java
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/java/quarkus/qute/SomePage.java
@@ -3,11 +3,11 @@ package quarkus.qute;
 import io.quarkus.qute.Template;
 import io.quarkus.qute.TemplateInstance;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
-import javax.ws.rs.core.MediaType;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.MediaType;
 
 import static java.util.Objects.requireNonNull;
 
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/resources/application.properties b/apps/quarkus-mp-orm-dbs-awt/src/main/resources/application.properties
index 9962bb7..d17f01b 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/resources/application.properties
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/resources/application.properties
@@ -39,11 +39,12 @@ mp.jwt.verify.publickey.location=META-INF/resources/publicKey.pem
 mp.jwt.verify.issuer=https://server.example.com
 quarkus.smallrye-jwt.enabled=true
 
-# Jaeger Q 2.x
-quarkus.jaeger.service-name=Demo-Service-A
-quarkus.jaeger.sampler-type=const
-quarkus.jaeger.sampler-param=1
-quarkus.jaeger.endpoint=http://localhost:14268/api/traces
+# OpenTelemetry Q 3.x
+quarkus.otel.traces.exporter=cdi
+quarkus.otel.traces.sampler=parentbased_always_on
+quarkus.otel.service.name=Demo-Service-A
+quarkus.otel.exporter.otlp.endpoint=http://localhost:4317
+quarkus.otel.exporter.otlp.traces.endpoint=http://localhost:4317
 
 # Additional native resources to include
 quarkus.native.resources.includes=privateKey.pem,MyFreeMono.ttf,MyFreeSerif.ttf,quarkus-icon.png
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/resources/importDB1.sql b/apps/quarkus-mp-orm-dbs-awt/src/main/resources/importDB1.sql
index dac4877..0c1154e 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/resources/importDB1.sql
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/resources/importDB1.sql
@@ -1,3 +1,4 @@
-insert into db1entity (id, field) values(nextval('hibernate_sequence'), 'field-1');
-insert into db1entity (id, field) values(nextval('hibernate_sequence'), 'field-2');
-insert into db1entity (id, field) values(nextval('hibernate_sequence'), 'field-3');
+-- Postgres
+insert into db1entity (id, field) values(nextval('db1entity_SEQ'), 'field-1');
+insert into db1entity (id, field) values(nextval('db1entity_SEQ'), 'field-2');
+insert into db1entity (id, field) values(nextval('db1entity_SEQ'), 'field-3');
diff --git a/apps/quarkus-mp-orm-dbs-awt/src/main/resources/importDB2.sql b/apps/quarkus-mp-orm-dbs-awt/src/main/resources/importDB2.sql
index b5f16ea..f0dcc9f 100644
--- a/apps/quarkus-mp-orm-dbs-awt/src/main/resources/importDB2.sql
+++ b/apps/quarkus-mp-orm-dbs-awt/src/main/resources/importDB2.sql
@@ -1,5 +1,5 @@
+-- MariaDB
 insert into db2entity (id, field) values(10, 'field-1');
 insert into db2entity (id, field) values(20, 'field-2');
 insert into db2entity (id, field) values(30, 'field-3');
 ALTER TABLE `db2entity` AUTO_INCREMENT = 40;
-
