diff --git "a/apps/quarkus-sp\303\266klik-encoding/pom.xml" "b/apps/quarkus-sp\303\266klik-encoding/pom.xml"
index 9fffa31..0bda3d3 100644
--- "a/apps/quarkus-sp\303\266klik-encoding/pom.xml"
+++ "b/apps/quarkus-sp\303\266klik-encoding/pom.xml"
@@ -42,6 +42,7 @@
         <groupId>io.quarkus</groupId>
         <artifactId>quarkus-maven-plugin</artifactId>
         <version>${quarkus.version}</version>
+        <extensions>true</extensions>
         <executions>
           <execution>
             <goals>
